test_unstructured/metrics/test_element_type.py

```diff
@@ -0,0 +1,87 @@
+import pytest
+
+from unstructured.metrics.element_type import get_element_type_frequency
+from unstructured.partition.auto import partition
+
+
+@pytest.mark.parametrize(
+    ("filename", "frequency"),
+    [
+        (
+            "fake-email.txt",
+            {
+                "UncategorizedText": [("None", 6)],
```

Comment:
I meant a tuple as the key in the dictionary of frequencies.
ex. `("UncategorizedText", "None"): 6` or ex. `("Title", 1): 3`

the type of the depth as string or numerical (to cover both None and 0, 1, etc) is up to you

Reply:
But in that case, how would the Element types that aren't exist in the doc be initialize? `("UncategorizedText",)` like this?

Reply:
In this way, each tuple key to the dictionary represents a unique element type. Overall, the type of the dictionary will be `Dict[Tuple[str, int], int]` (or possibly str for second value in tuple to capture None)

Would it then no longer be possible to create all the keys for the dictionary? Probably. Let's instead prioritize having meaningful keys instead of initializing the dictionary with all elements categories.

Reply:
So we revert back to not having all the Element Type initialized, then assign value to each tuple key.